File: miguel/(bernardo)/Problema0211/problema0211.java
import java.util.Scanner;

public class problema0211 {
    static Scanner input = new Scanner(System.in);

    public static void main(String[] args) {

        // Declaração de variáveis
        String ingresso;
        int socio = 0;
        int naoSocio = 0;
        float percentual = 0;
        int ingressos;
        int valorSocios;
        int valorNaoSocios;
        int valorTotal;

        // Entrada de valores e contagem de ingressos
        for (var i = 0; i < 10; i++) {
            System.out.println(
                    "Digite o tipo do " + (i + 1) + " ingresso(socio ou nao-socio) ou digite fim para finalizar: ");
            ingresso = String.valueOf(input.nextLine());
            if (ingresso.toLowerCase().equals("socio")) {
                socio++;
            } else if (ingresso.toLowerCase().equals("nao-socio")) {
                naoSocio++;
            } else if (ingresso.toLowerCase().equals("fim")) {
                break;
            } else {
                System.out.println("Entrada " + ingresso + " não é reconhecida");
                break;
            }
        }

        // Cálculos
        percentual = 0;
        ingressos = socio + naoSocio;
        if (ingressos > 0) {
            percentual = (socio * 100) / ingressos;
        }
        valorSocios = 50 * socio;
        valorNaoSocios = 100 * naoSocio;
        valorTotal = valorSocios + valorNaoSocios;

        // Saída
        System.out.println("A quantidade de ingressos para sócios é: " + socio);
        System.out.println("A quantidade de ingressos para não_sócios é: " + naoSocio);
        System.out.println("O percentual de ingressos para sócios é: " + percentual + "%");
        System.out.println("O valor recebido de sócios é: R$" + Math.round(valorSocios * 100) / 100);
        System.out.println("O valor recebido de não-sócios é: R$ " + Math.round(valorNaoSocios * 100) / 100);
        System.out.println("O valor total recebido é: R$" + Math.round(valorTotal * 100) / 100);
    }
}

File: miguel/(bernardo)/Problema0211/problema0211.py
# Entrada de valores e contagem de ingressos
socio = 0
nao_socio = 0
for n in range(10):
    ingresso = str(input(
        f"Digite o tipo do {n+1} ingresso(socio ou nao-socio) ou digite fim para finalizar: "))
    if ingresso.lower() == "socio":
        socio += 1
    elif ingresso.lower() == "nao-socio":
        nao_socio += 1
    elif ingresso.lower() == "fim":
        break
    else:
        print(f"Entrada {ingresso} não é reconhecida")
        break

# Cálculos
percentual = 0
ingressos = socio + nao_socio
if ingressos > 0:
    percentual = (socio * 100) / ingressos
valor_socios = 50 * socio
valor_nao_socios = 100 * nao_socio
valor_total = valor_socios + valor_nao_socios

# Saída
print(f"A quantidade de ingressos para sócios é: {socio}")
print(f"A quantidade de ingressos para não_sócios é: {nao_socio}")
print(f"O percentual de ingressos para sócios é: {percentual} %")
print(f"O valor recebido de sócios é: R${round(valor_socios,2)}")
print(f"O valor recebido de não-sócios é: R${round(valor_nao_socios,2)}")
print(f"O valor total recebido é: R${round(valor_total,2)}")
